Cargo.toml

```diff
@@ -11,3 +11,4 @@ incremental = true
 debug = 0 # set this to 1 or 2 to get more useful backtraces in debugger
 
 [patch.'crates-io']
+# rowan = { path = "../rowan" }
```

Comment:
should this line be removed?

Reply:
Nope: I always forget the patchin syntax, and this helps

Reply:
ah, understood